src/vim9execute.c

```diff
@@ -1998,7 +1998,7 @@ ex_disassemble(exarg_T *eap)
 			    when = "JUMP_AND_KEEP_IF_FALSE";
 			    break;
 		    }
-		    smsg("%4d %s -> %lld", current, when,
+		    smsg("%4d %s -> %d", current, when,
```

Comment:
Thanks!
I wonder though why it was not caught at compilation time since `smsg` is declared as:
```
int smsg(const char *, ...)
# ifdef USE_PRINTF_FORMAT_ATTRIBUTE
    __attribute__((format(printf, 1, 0)))
# endif
    ;
```